UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -76,6 +79,7 @@ public async Task QueryAndDownloadFileShareServiceFiles(SalesCatalogueServiceRes
         }
         private async Task CreateAncillaryFiles(string batchId, string exchangeSetRootPath, string correlationId)
```

Comment:
Change parameter name passed from exchangeSetRootPath to exchangeSetPath as the calling method parameter name is changed to exchangeSetPath. Also define one variable var exchangeSetRootPath = Path.Combine(exchangeSetPath, fileShareServiceConfig.Value.EncRoot);

Reply:
Resolved